fala/templates/base.html

```diff
@@ -66,10 +66,7 @@ <h1>
     {% block search_form %}{% endblock %}
 
     {% if not data.count %}
-      <p>
-        For advice on education or discrimination contact
-        <a href="https://www.gov.uk/civil-legal-advice" rel="external" target="_blank">Civil Legal Advice.</a>
-      </p>
+    
```

Comment:
Yep, as you wondered, there's no point in the `{% if ... %}` and `{% endif %}` statements now, so they should be removed too.

Reply:
Thanks @exonian - i've removed the if statements. Can you look again?